package.json

```diff
@@ -35,18 +35,18 @@
     "onLanguage:typescriptreact",
     "onLanguage:javascript",
     "onLanguage:javascriptreact",
-    "onCommand:deno.cache",
-    "onCommand:deno.status",
+    "onCommand:deno.welcome",
+    "onCommand:deno.initializeWorkspace",
     "onWebviewPanel:welcomeDeno"
   ],
   "main": "./client/out/extension",
   "contributes": {
     "commands": [
       {
-        "command": "deno.cache",
-        "title": "Cache Dependencies",
+        "command": "deno.welcome",
```

Comment:
Why are you reordering these, they were ordered alphabetical versus arbitrary.

Reply:
Reverted.